packages/fast-components-react-base/src/toggle/toggle.spec.tsx

```diff
@@ -38,68 +38,80 @@ describe("toggle", (): void => {
         statusLabelId: "statusLabelId",
         unselectedString: "Unselected"
     };
+    const inputSelector: string = `.${managedClasses.toggle_input}`;
 
-    test("should return an object that includes all valid props which are not enumerated as handledProps", () => {
+    test("should implement unhandledProps", () => {
         const unhandledProps: IToggleUnhandledProps = {
-            "aria-hidden": true
+            "data-my-custom-attribute": true
         };
         const props: ToggleProps = {...handledProps, ...unhandledProps};
         const rendered: any = shallow(
             <Component {...props} />
         );
 
-        expect(rendered.prop("aria-hidden")).not.toBe(undefined);
-        expect(rendered.prop("aria-hidden")).toEqual(true);
+        expect(rendered.first().prop("data-my-custom-attribute")).toBe(true);
     });
 
-    test("should correctly operate as an uncontrolled component and set initial state if `selected` prop is not passed", () => {
+    test("should initalize as unselected if the `selected` prop is not provided", () => {
         const rendered: any = shallow(
             <Component {...handledProps} />
         );
 
-        const state: any = rendered.state("checked");
-
-        expect(state).toBe(undefined);
+        expect(rendered.state("selected")).toBe(false);
     });
 
-    test("should correctly operate as an uncontrolled component and handle `onChange` events when `selected` prop is not passed", () => {
+    test("a change event should change the selected state when no `selected` prop is not provided", () => {
```

Comment:
nit: can we re-word to read more like "should allow a change event to update the selected state when no `selected` prop is provided" so that our tests can read more like a sentence when starting with the test suites describe string "toggle". This is also a double negative so was a little confusing to read.